languages/java/exercises/concept/case-statement/src/test/java/PlayAnalyzerTest.java

```diff
@@ -0,0 +1,58 @@
+import org.junit.Assert;
+import org.junit.Test;
+
+public class PlayAnalyzerTest {
+
+    @Test(expected = IllegalArgumentException.class)
+    public void test_exception(){
+        PlayAnalyzer.analyseOnField(13);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
All but the first test should be marked with `@Ignore("Remove to run test")`, as we want students to focus on solving tests one at a time. See [this existing Concept Exercise's tests](https://github.com/exercism/v3/blob/master/languages/java/exercises/concept/basics/src/test/java/LasagnaTest.java#L13).